test/functionalTest/harnessFunctions.sh

```diff
@@ -595,22 +625,200 @@ function dbInsertEntity()
 #   --host        <host>           (default: localhost)
 #   --port        <port>           (default: $CB_PORT)
 #   --url         <URL>            (default: empty string)
+#   --urlParams   <params>         (URI parameters 'in' URL-string)
 #   --payload     <payload>        (default: NO PAYLOAD. Possible values: [filename | "${string}"])
 #   --in          (input payload)  (default: xml => application/xml, If 'json': application/json)
 #   --out         (output payload  (default: xml => application/xml, If 'json': application/json)
 #   --json        (in/out JSON)    (if --in/out is used AFTER --json, it overrides) 
 #   --httpTenant  <tenant>         (tenant in HTTP header)
+#   --tenant      <tenant>         (tenant in HTTP header)
```

Comment:
I would like to migrate from --httpTenant to --tenant.
--httpTenant is an old name, from when we had both --httpTenant and --urlTenant.
It's name should have changed when --urlTenant was removed.
Now orionCurl is prepared for this change.
I need to create an issue about this ...